src/app/personside/infotabs/meldinger/utils/meldingerUtils.test.ts

```diff
@@ -56,18 +56,18 @@ describe('Meldingstyper', () => {
 describe('Dokumentvarsler', () => {
     const tomTemaGruppeNull = null;
     const tomTemaGruppeEmpty = '';
-    const tomTemagruppeTNull = Temagruppe.Null;
+    const tomTemagruppeSlettet = Temagruppe.InnholdSlettet;
 
     it('Gir temagruppe Arbeid ved temagruppe ARB', function() {
         expect(temagruppeTekst(Temagruppe.Arbeid)).toBe('Arbeid');
     });
     it('Gir tom temagruppe på dokumentvarsler med null', function() {
         expect(temagruppeTekst(<Temagruppe>(<unknown>tomTemaGruppeNull))).toBe('');
     });
-    it('Gir tom temagruppe på dokumentvarsler med emtpy', function() {
-        expect(temagruppeTekst(<Temagruppe>tomTemaGruppeEmpty)).toBe('');
+    it('Gir riktig temagruppe på dokumentvarsler med emtpy (da dette tolkes som slettet melding)', function() {
+        expect(temagruppeTekst(<Temagruppe>tomTemaGruppeEmpty)).toBe('Innhold slettet');
     });
-    it('Gir tom temagruppe på dokumentvarsler temagryppe.Null', function() {
-        expect(temagruppeTekst(<Temagruppe>tomTemagruppeTNull)).toBe('');
+    it('Gir riktig temagruppe på dokumentvarsler temagruppe.Slettet', function() {
+        expect(temagruppeTekst(<Temagruppe>tomTemagruppeSlettet)).toBe('Innhold slettet');
     });
```

Comment:
Litt usikker på desse to, då innhold slettet skal ikkje komme opp på dokumentvarsler, men ein standardtekst frå baksystemet om kva melding dette er.

Reply:
Ok, da må vi se på om dokumentvarsler har tom string som temagruppe. Isåfall må logikken her bli litt mer komplisert